lib/libc/minimal/include/sys/types.h

```diff
@@ -16,6 +16,10 @@ typedef unsigned int mode_t;
 #if !defined(__ssize_t_defined)
 #define __ssize_t_defined
 
+/* parasoft suppress item MISRAC2012-RULE_20_4-a item MISRAC2012-RULE_20_4-b
```

Comment:
> Add a parasoft tags to suppress a violation in static analysis tool
Is this supposed to be in the upstream?

Reply:
yes. where else?

Reply:
This `parasoft` thing seemed to be the first of its kind, so I was just wondering if he mistakenly submitted something intended for private branch to upstream.

Reply:
parasoft is the [official tool](https://www.parasoft.com/news/the-zephyr-project-selects-parasofts-functional-safety-compliance-solution-to-validate-safety/) the project uses for static analysis and coding guideline compliance.

Reply:
> parasoft is the official tool the project uses for static analysis and coding guideline compliance.

Interesting. It would be nice to have this documented somewhere (especially the policy regarding the use of these "parasoft tags").

Reply:
@nashif Should I prepare wiki page about Parasoft and tags for it?